apps/docs/docs/open-api.mdx

```diff
@@ -0,0 +1,140 @@
+import { InstallTabs } from '@site/src/components/InstallTabs';
+
+# Open API
+
+## Installation
+
+Much like the other packages, `@ts-rest/open-api` is easy to download, also note if you want to serve a Swagger UI you'll want to download `swagger-ui-express` for express, or `@nestjs/swagger ` for Nest.
+
+<InstallTabs packageName="@ts-rest/open-api" />
+
+## Generating a OpenAPI Document
+
+```typescript
+import { myContract } from './my-api';
+import { generateOpenApi } from '@ts-rest/open-api';
+
+const openApiDocument = generateOpenApi(myContract, {
+  info: {
+    title: 'Posts API',
+    version: '1.0.0',
+  },
+});
+```
+
+No kidding, that's it 🥳 You can now serve this document however you want, use it for CodeGen for your non-TS clients, or use it to generate a Swagger UI!
+
+:::info
+Our OpenAPI generator uses `@anatine/zod-openapi` to geneate open-api docs for your Zod schemas, so make sure to use Zod everywhere on your API to get the best experience.
+:::
+
+## Serving a Swagger UI
+
+In Express use `swagger-ui-express`:
```

Comment:
We can use tabs (Express, NestJS) for a nicer UX like we have on the Quickstart page